src/internal.c

```diff
@@ -13656,7 +13714,8 @@ int ProcessPeerCerts(WOLFSSL* ssl, byte* input, word32* inOutIdx,
                     }
                 #endif /* SESSION_CERTS && WOLFSSL_ALT_CERT_CHAINS */
 
-                #ifndef OPENSSL_COMPATIBLE_DEFAULTS
+                #if !defined(OPENSSL_COMPATIBLE_DEFAULTS) && \
+                    !defined(DISABLE_PEER_CERT_VERS_CHECK)
```

Comment:
Please clearly document `DISABLE_PEER_CERT_VERS_CHECK`

Reply:
Removed.  It is guarding code that is only executed on the server side.